frontend/src/Buildings/addBuildingItem.jsx

```diff
@@ -21,14 +22,22 @@ const styles = StyleSheet.create({
     fontSize: FontStyle.fontSize,
     fontWeight: 'bold',
   },
+  subText: {
+    marginLeft: 8,
+    fontSize: 10,
+    color: Colors.orangeColor,
+  },
 });
 
 function AddBuildingItem({ icon, type, onPress }) {
   return (
     <TouchableHighlight underlayColor="transparent" onPress={onPress}>
       <View style={styles.itemStyle}>
         <Image style={styles.iconStyle} source={icon} />
-        <Text style={styles.textStyle}>{`Add ${type}`}</Text>
+        <View>
+          <Text style={styles.textStyle}>{`Add ${type}`}</Text>
+          <Text style={styles.subText}>-100 golds</Text>
```

Comment:
cost should come from the rules (`constructionCost`)

Reply:
If I use the constructioncost, I have to import at the getBuilding part. Can I touch the get building information part?